internal/types/kruizePayload/common.go

```diff
@@ -256,3 +281,23 @@ func make_container_data(c map[string]interface{}) container {
 
 	return container_data
 }
+
+func MemBytesToMi(mem float64) float64 {
+	memoryInMi := mem / 1024 / 1024
+	truncatedMem := math.Trunc(memoryInMi*100) / 100
+	return truncatedMem
+}
+
+func isMemoryField(metricName string) bool {
+	// TO DO: In go 1.21 use slices package instead of map
+	memoryFields := map[string]string{"memoryRSS": "", "memoryUsage": "", "memoryLimit": "", "memoryRequest": ""}
+	_, keyExists := memoryFields["metricName"]
```

Comment:
I see some bad memory values on API response, maybe this?
Seems like `metricName string` is already a string.